pinot-common/src/main/java/org/apache/pinot/common/function/FunctionRegistry.java

```diff
@@ -79,6 +79,9 @@ private FunctionRegistry() {
         }
       }
     }
+    for (Method method : PlaceholderScalarFunctions.class.getMethods()) {
```

Comment:
thanks for this PR! can you add some test cases to `QueryRunnerTest` to validate that functions registered in this way are picked up end-to-end?

Reply:
yup wilco